app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  validates :name, :email, :address, :description, presence: true
+  validates_format_of :email, with: Devise::email_regexp
+  has_many :venues, dependent: :destroy
+  has_many :bookings, dependent: :destroy
+end
```

Comment:
no dependent destroy as the owner still needs access to the booking if the user is gone